tests/ci_build/Dockerfile.build_cuda

```diff
@@ -1,4 +1,4 @@
-FROM nvidia/cuda:8.0-cudnn7-devel
```

Comment:
Maybe a typo?  Looks like you created a separate Dockerfile for cudnn7 right?

Reply:
Yeah, it was originally cudnn5 and I changed it in hope of running cudnn7, but wasn't successful. So I changed it back and Marco helped me setup another docker file with cudnn7.